apps/aehttpclient/lib/client.ex

```diff
@@ -95,6 +95,20 @@ defmodule Aehttpclient.Client do
     get(uri <> "/tx_pool/#{acc}", :acc_txs)
   end
 
+  def post_query_to_oracle(tx, uri) do
```

Comment:
Why do we treat query txs specially? Shouldn't oracle react to query when it's included in block?

Reply:
This is old code I forgot to remove